bundle/config/mutator/translate_paths.go

```diff
@@ -332,7 +332,7 @@ func gatherFallbackPaths(v dyn.Value, typ string) (map[string]string, error) {
 	// the original approach as fallback if the [dyn.Value] location cannot be resolved.
```

Comment:
This probably no longer works now that the path has been normalized.

I'm OK removing this entirely for the sake of simplicity, but would need to assess the impact.